src/src_V2/app/App.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {BrowserRouter as Router, useRoutes,} from "react-router-dom";
```

Comment:
spacing, break line, and pls remove last comma.

Reply:
Yeah sure, but i think we also need some eslint configs, for avoiding this kinda cases.

Reply:
Yes, I'm agree